src/main/java/net/dv8tion/jda/api/events/user/update/UserUpdateAccentColorEvent.java

```diff
@@ -0,0 +1,50 @@
+package net.dv8tion.jda.api.events.user.update;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.User;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.awt.*;
+
+/**
+ * Indicates that the accent color of a {@link net.dv8tion.jda.api.entities.User User} changed.
+ *
+ * <p>Can be used to retrieve the user who changed their accent color and their previous accent color.
+ *
+ * <p>Identifier: {@code accent_color}
+ *
+ * <h2>Requirements</h2>
+ * <p>This event requires that the user be fetched by the {@link net.dv8tion.jda.api.JDA#retrieveUserById(long)} method
```

Comment:
But `retrieveUserById` will not add the user to cache

Reply:
It seems retrieving them will fire the update since the entity builder method is called.

Reply:
My point is that retrieve will not add it to cache, if its not cached it won't fire an update event.